css/style.css

```diff
@@ -1,11 +1,159 @@
 
  /* Use the 'Roboto' font from Google. Already added to your HTML header */
 
+ body {
+  font-family: 'Roboto', sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
+
+.nav-bar {
+  background: #fff;
+  position: fixed;
+  width: 100%;
+  border-bottom: 1px solid #DDD;
+  z-index: 9999;
+ 
+}
+
+.nav-bar .links {
+  height: 5.35rem;
+  width: 60%;
+  display: flex;
+  justify-content: center;
+  margin-top: 1.5rem;
+}
+
+.links {
+  padding: 0 1rem;
+  margin: 0 auto;
+  
+}
+
+.nav-bar .links li {
+  display: inline-flex;
+  color: #504d4b;
+  font-size: 1.5rem;
+  margin-left: 1.3rem;
+}
+
+.nav-bar .logo {
+  height: 3.45rem;
+}
+
+.main {
+  background-image: url('../img/first-background.jpg');
+  background-position: center bottom;
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 50.35rem;
+  width: auto;
+  padding: 20rem 5rem 5rem 3rem;
```

Comment:
You don't need this! Remove width, padding and margin and use your `col-` classes instead.